src/main/java/gregtech/core/CoreModule.java

```diff
@@ -164,6 +167,9 @@ public void preInit(FMLPreInitializationEvent event) {
 
         proxy.onPreLoad();
         KeyBind.init();
+
+        LDItemPipeType.INSTANCE.getName();
+        LDFluidPipeType.INSTANCE.getName();
```

Comment:
I feel like we should find a better way to do this, even if its just calling an `init()` method on both of these classes that creates the static INSTANCE field for each